src/js/Dialog/__test__/rerender.test.ts

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable @typescript-eslint/no-empty-function */
+
+import Dialog from '../index';
+
+describe('Unit test Dialog render', () => {
+
+  beforeEach(() => {
+    jest.spyOn(console, 'error');
+    // @ts-ignore
+    console.error.mockImplementation(() => {});
+  });
+  afterEach(() => {
+    // @ts-ignore
+    console.error.mockRestore();
+  });
+
+  test('Function rerender work normally with no params', () => {
+    const myDialog = new Dialog({});
+    try {
+      myDialog.rerender();
+      expect(true);
+    } catch (error) {
+      if (error) {
+        expect(false);
+      }
+    }
```

Comment:
Should change to:
try {
      myDialog.rerender();
    } catch (error) {
        expect(false).toEqual(true);
    }